docs/Reference/JSON-RPC-API-Methods.md

```diff
@@ -817,20 +817,18 @@ You can interact with contracts using [eth_sendRawTransaction or eth_call](../Us
 
 ### eth_estimateGas
 
-Returns an estimate of how much gas is needed for the transaction to complete. The estimate process does not use
-gas and is not added to the blockchain as a transaction. The estimate can be greater than the amount of gas that the
-transaction actually uses, for reasons including EVM mechanics and node performance.
+Returns an estimate of how much gas is needed for a future transaction to complete. The estimation process does not use
```

Comment:
I think 'future' is redundant and could be removed (it's covered in the following sentence).

Reply:
Deleted 'future'.